js/MVP/globals/constants.js

```diff
@@ -16,3 +16,9 @@ export const pdfTypes = {
   REFERAT: 'referat',
   BREV: 'brev',
 };
+
+export const svarInnkallelseTypes = {
```

Comment:
Vi burde egentlig ha dette som en del av typedefinisjon for api-kall mot isdialogmote.

Jeg kan godt hjelpe med dette om du vil, eller om du vil droppe det nå så kan vi vente med det til en annen gang

Reply:
Jeg har lagt inn typing av response fra isdialogmote her: https://github.com/navikt/dialogmote/pull/117
(Siden det har vært litt usikkerhet rundt responsknapper tenkte jeg at det kunne være greit å ha i en egen PR.

Reply:
Har lagt inn typing av svar i hht det som er planlagt i isdialogmote: https://github.com/navikt/isdialogmote/pull/222/files